gitpod-io/gitpod
# [Observability] Add node resource consumption dashboards

## Description

Add grafana dashboard with id [11074](https://grafana.com/grafana/dashboards/11074/revisions) and update it to include cluster
level filtering and translation of chinese language to english.

## Related Issue(s)

Fixes https://github.com/gitpod-io/gitpod/issues/7645

## How to test

[Import](https://grafana.com/docs/grafana/latest/dashboards/export-import/#import-dashboard) the content of the json file as a grafana dashboard. You should be able to see the dashboard.

e.g. https://grafana.gitpod.io/d/xfpJB9FGz6/example-node-resource-usage-metrics?orgId=1
## Release Notes

```release-note
None
```

## Documentation
